packages/components/back-top/src/BackTop.vue

```diff
@@ -15,10 +15,10 @@ import type { BackTopProps } from './types'
 import { defineComponent, nextTick, onUnmounted, onMounted, ref } from 'vue'
 import throttle from 'lodash/throttle'
 import { IxIcon } from '@idux/components/icon'
-import { PropTypes, withUndefined, isString, isHTMLElement, on, off, rAF } from '@idux/cdk/utils'
+import { PropTypes, withUndefined, on, off, rAF } from '@idux/cdk/utils'
 import { easeInOutQuad } from '@idux/components/core/utils'
 import { useGlobalConfig } from '@idux/components/core/config'
-import { Logger } from '@idux/components/core/logger'
+import { getTarget } from '@idux/components/core/utils/propsHandler'
```

Comment:
```suggestion
import { getTarget } from '@idux/components/core/utils'
```

跟上面的 easeInOutQuad  合到一起吧